credentialdigger/scanners/git_scanner.py

```diff
@@ -48,32 +49,30 @@ def stream(self, rules):
                              elements=len(patterns),
                              flags=flags)
 
-    def scan(self, git_url, since_commit=None, max_depth=1000000):
+    def scan(self, git_url, since_timestamp=0, max_depth=1000000):
         """ Scan a repository.
 
         Parameters
         ----------
         git_url: string
             The url of a git repository
-        since_commit: string, optional
-            The oldest commit to scan
+        since_timestamp: int, optional
+            The oldest timestamp to scan
         max_depth: int, optional
             The maximum number of commits to scan
 
         Returns
         -------
-        str
-            The latest commit (`None` if the repository is empty)
+        int
+            The latest scan timestamp (now) (`None` if the repository is empty)
```

Comment:
`0` if the repository is empty (that None comes from the old method)

Reply:
The method still returns `None` if the repository is empty, as you can see here:
https://github.com/SAP/credential-digger/blob/ba4d7b7b61c176e6e8b7d9187f398a3e06c1a10a/credentialdigger/scanners/git_scanner.py#L75-L77

Should it return 0 instead? In that case we wouldn't be able to know if the repo has 0 discoveries or is simply empty, but I don't think we need to differentiate the two cases.

Reply:
No, you're right, let's leave it like it is. I misunderstood the code